app/client/src/utils/rules.ts

```diff
@@ -0,0 +1,2 @@
+export const QUESTIONS_MAX_LENGTH = 1000
+export const FEEDBACK_MAX_LENGTH = 1000
```

Comment:
These could possibly be switched to env variables.

Reply:
Ah, I told him to make it this way. https://github.com/c2c-project/prytaneum/pull/259#discussion_r812574922